src/main/java/com/example/gruppajava/controller/ParkingEventController.java

```diff
@@ -0,0 +1,81 @@
+package com.example.gruppajava.controller;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.gruppajava.entity.ParkingEvent;
+import com.example.gruppajava.repository.CarRepository;
+import com.example.gruppajava.repository.ParkingEventRepository;
+import com.example.gruppajava.repository.UserRepository;
+
+
+@RestController
+@RequestMapping("/api/")
+public class ParkingEventController {
+    @Autowired ParkingEventRepository parkingEventRepository;
+    @Autowired CarRepository carRepository;
+    @Autowired UserRepository userRepository;
+
```

Comment:
Autowiring these seems like a smart move, less code than using a constructor like me and Marcus 👍

Reply:
Injection power 💪